providers/src/airflow/providers/amazon/aws/utils/waiter.py

```diff
@@ -83,3 +84,16 @@ def get_state(response, keys) -> str:
         if value is not None:
             value = value.get(key, None)
     return value
+
+
+class WaitPolicy(str, Enum):
+    # Wait for the cluster to be up.
+    DEFAULT = "default"
```

Comment:
Lets not say default but be explicit about what it is.
I think we have 
`wait_for_completion` - equivalent to what we have now
`wait_for_completion_with_machine_terminated` - the new functionality you wanted

You can suggest better names

Reply:
That's good.

Reply:
Done